modules/tinymce/src/themes/silver/main/ts/ui/promotion/Promotion.ts

```diff
@@ -0,0 +1,45 @@
+import { AlloySpec, SimpleSpec } from '@ephox/alloy';
+
+import Editor from 'tinymce/core/api/Editor';
+
+import { UiFactoryBackstageShared } from '../../backstage/Backstage';
+
+const promotionMessage = 'Try Premium!';
+const promotionLink = 'http://tiny.cloud';
+
+export interface PromotionSpec extends SimpleSpec {
+  editor: Editor;
+  sharedBackstage: UiFactoryBackstageShared;
+}
```

Comment:
Why do we need this? It's never used below is all 🤷

Reply:
Because I got distracted trying to get the tests to pass and forgot to delete it. Fixed.